weasyl/templates/help/community.html

```diff
@@ -69,17 +78,23 @@ <h4>C. Prohibited Upload Material</h4>
 
       <ol>
         <li><strong>Offensive Content:</strong> Submissions should avoid expressing bigoted or hateful material (e.g. racist, sexist, homophobic, etc.) to the extent possible without compromising the artistic integrity of the piece.  Material that appears intended for the sole purpose of provoking hostile responses is not permitted.</li>
-        <li><strong>Minors in Sexual/Mature Situations:</strong> Submissions and other uploaded content must not contain characters who are or are depicted as being under the age of eighteen present in sexual situations, whether or not they are directly involved. Additionally, minors may not be depicted with unacceptable nudity. "Depicted as being under the age of eighteen" can mean one of two things:
-          <ul style="padding-top: 1.5em;">
-            <li>The character is referred to or implied as being under 18, whether in a visual or written piece, or;</li>
-            <li>The character in question has small child- or infant-like proportions (excepting stylized depictions of adults that incorporate childlike proportions, e.g. “chibi”). Canon is taken into consideration; however, if the character has the appearance of being under 18 or is not depicted “aged up”, the submission may still be removed.</li>
-          </ul>
-          <br />Unacceptable nudity for the purposes of this article entails both of the following:
+        <li><strong>Young-Looking (“Cub”) Characters and Allowed Content Ratings:</strong> A character is  considered a “Cub” by meeting at least one of these two criteria:
           <ul style="padding-top: 1.5em;">
-            <li>Genitals, buttocks, and/or mammaries with nipples are viewable.</li>
-            <li>Not depicted using non-detailed nudity or cartoon nudity.</li>
+            <li>They are expressly described in the submission as being under the age of 18 (e.g., in dialogue, the caption or description), or;</li>
```

Comment:
This would seem to create a loophole: draw a character underage and say they're over 18...